database/migrations/settings/2021_09_30_100000_create_katapult_settings.php

```diff
@@ -8,6 +8,7 @@ public function up(): void
     {
         $this->migrator->add('katapult.enabled', false);
         $this->migrator->add('katapult.public_key', '');
+        $this->migrator->add('katapult.private_key', '');
```

Comment:
Adding the new column to the database both in the existing migration and in a new migration will cause it to thrown an error on new installs since when it tries to migrate `database/migrations/settings/2022_06_24_10000_add_private_key.php` the `katapult.private_key` is already there.

Instead leave this file untouched and only keep the `database/migrations/settings/2022_06_24_10000_add_private_key.php` file.